package.json

```diff
@@ -66,6 +66,7 @@
     "execa": "^6.0.0",
     "filter-obj": "^5.0.0",
     "find-up": "^6.0.0",
+    "get-tsconfig": "^4.6.2",
```

Comment:
nice, this package looks like it's solving sooo much for us! I got a bit suspicious because it says in the README that it "validates" the tsconfig (which we wouldn't want to do here, I think) - but I couldn't find anything about that in the code.